src/components/GlobalPreferences/GlobalPreferencesButton.js

```diff
@@ -84,17 +85,33 @@ function GlobalPreferencesButton({ onOpen }) {
           </li>
           <Item
             onClick={handleItemClick('network')}
+            iconFirst
             icon={iconNetwork}
             label="Network"
           />
+          <Item
+            iconFirst={false}
+            icon="IconExternal"
+            label="Any questions? Visit our FAQ"
+            href="https://help.aragon.org/article/48-aragon-court-faq"
+          />
         </ul>
       </Popover>
     </React.Fragment>
   )
 }
 
-function Item({ icon, label, onClick }) {
+function Item({ icon, iconFirst, label, onClick, href }) {
   const theme = useTheme()
+  let Icon
+
+  switch (icon) {
```

Comment:
Let's update the old API as we find new edge cases :).

Rather than doing this, let's force users to pass down the node they'd like to render in the icon slot (e.g. users would pass down `IconExternal` or `<img src={icon} />`).